docker-compose.yml

```diff
@@ -143,6 +143,16 @@ services:
       - RABBITMQ_USER=${RABBITMQ_DEFAULT_USER}
       - RABBITMQ_PASSWORD=${RABBITMQ_DEFAULT_PASS}
       - REPORT_BASE_URL
+      - UPLOADER_BASE_URL
+      - UPLOADER_CLIENT_ID
+      - EXPOSE_BASE_URL
+      - UPLOADER_CLIENT_RANDOM_ID
+      - UPLOADER_CLIENT_SECRET
+      - EXPOSE_CLIENT_ID
+      - EXPOSE_CLIENT_RANDOM_ID
+      - EXPOSE_CLIENT_SECRET
+      - DEFAULT_USER_EMAIL
+      - DEFAULT_USER_PASSWORD
```

Comment:
I'd prefer to not have this permanently in the container. It's only useful for the first setup (I insist for an external job :))